### DISCUSSION TITLE
Last V9 Release contains deprecated types.

### DISCUSSION THREAD

**Discussion by Chibas** [pact-foundation/pact-js#978] [Q&A]
I'm stuck with pact v9 because of amount of tests to rewrite.
After updating to typescript v4 build started failing with errors
```
error TS2688: Cannot find type definition file for 'pino-pretty'.
  The file is in the program because:
    Entry point for implicit type library 'pino-pretty'

error TS2688: Cannot find type definition file for 'pino-std-serializers'.
  The file is in the program because:
    Entry point for implicit type library 'pino-std-serializers'
```

It seems that all versions prior to 10 has a dependency @types/pino with an old deprecated types for TS v3. Deleting node_modules/@types/pino or using overrides value  `@types/pino": "" `resolves the issue and it starts to use pino's own types.
Is there a non hacky way to use v9 with TS v4 ?

**Comment by mefellows** [comment]
It looks related to [#types](https://github.com/pact-foundation/pact-js-core/issues/417).
